src/BinaryParsers/BinaryParsers.csproj

```diff
@@ -5,6 +5,12 @@
     <RootNamespace>Microsoft.CodeAnalysis.BinaryParsers</RootNamespace>
     <TargetFramework>$(NetStandardVersion)</TargetFramework>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>

```

Comment:
Can collapse this into a single property group if you remove the condition attribute